### PR TITLE
fix(designer): do not set Panel header horizontal padding when Panel is collapsed

### PR DESCRIPTION
**Pull request by jaimereyesmx**
Before:
image

Now:
image

### CODE REVIEW

**Comment by ek68794998** [comment]
LGTM. This meets our bug bar since the double-wide panel blocks the minimap controls.

File: libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx [hunk 1]
```diff
@@ -226,7 +226,7 @@ export const PanelHeader = ({
       </div>
       {!noNodeOnCardLevel ? (
         <>
-          <div className={'msla-panel-card-header'} style={isRight ? {} : { paddingLeft: horizontalPadding }}>
+          <div className={'msla-panel-card-header'} style={isRight || isCollapsed ? {} : { paddingLeft: horizontalPadding }}>
```
Comment (preetriti1) [inline libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx:229]: @rllyy97 - Will this remove any kind of padding in our portal which is needed?

Comment (jaimereyesmx) [reply libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx:229]: Only the padding when panel is on the left that I added last week. No impact to other padding.

**Review by rllyy97 — APPROVED** [review]

File: libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx (hunk 1 above)
Comment (rllyy97) [reply libs/designer-ui/src/lib/panel/panelheader/panelheader.tsx:229]: Just tested and looks fine for us @preetriti1

**Pull request merged**